lib/open_food_network/permissions.rb

```diff
@@ -100,9 +90,9 @@ def visible_line_items
       editable = editable_line_items.pluck(:id)
 
       # Any from visible orders, where the product is produced by one of my managed producers
-      produced = Spree::LineItem.where(order_id: visible_orders.pluck(:id)).joins(:product).
-        where("spree_products.supplier_id IN (?)",
-              managed_enterprises.is_primary_producer.pluck(:id))
+      produced = Spree::LineItem.where(order_id: visible_orders.pluck(:id)).
```

Comment:
I started cleaning up up this file a little bit (there's one or two less queries triggered now, and it's a bit cleaner) but I realised it would be a much bigger job to get real results here, so I didn't want to delve any deeper. I've mentioned it elsewhere in related open issues.